samples/nrf9160/modem_shell/src/cloud/cloud_lwm2m.c

```diff
@@ -163,14 +175,12 @@ static int cloud_lwm2m_init(const struct device *unused)
 	ret = modem_info_init();
 	if (ret < 0) {
 		printk("LwM2M: Unable to init modem_info (%d)", ret);
-		return ret;
 	}
 
 	/* Query IMEI. */
 	ret = modem_info_string_get(MODEM_INFO_IMEI, imei_buf, sizeof(imei_buf));
 	if (ret < 0) {
 		printk("LwM2M: Unable to get IMEI");
-		return ret;
```

Comment:
How does the code work if we don't get IMEI and we use it in endpoint_name?

Reply:
And same here.